utils/process.py

```diff
@@ -0,0 +1,52 @@
+# stdlib
+import errno
+import os
+
+# 3p
+try:
+    import psutil
+except ImportError:
+    psutil = None
+
+# project
+from util import Platform
+
+
+def pid_exists(pid):
+    """
+    Check if a pid exists.
+    Lighter than psutil.pid_exists
+    """
+    if psutil:
+        return psutil.pid_exists(pid)
+
+    # We don't support windows for now
+    if Platform.is_windows():
+        raise NotImplementedError("pid_exists is not implemented for Windows")
```

Comment:
You should have a look into this:
https://github.com/DataDog/dd-agent/blob/146339a859ee81ed9b3d6d6fd19f101667d1aa3e/jmxfetch.py#L237-L248

It's a hacky but working way to check if a pid is running on windows